.ds.baseline

```diff
@@ -137,42 +137,6 @@
         "is_secret": false
       }
     ],
-    ".github/workflows/checks.yml": [
-      {
-        "type": "Secret Keyword",
-        "filename": ".github/workflows/checks.yml",
-        "hashed_secret": "5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8",
-        "is_verified": false,
-        "line_number": 28,
-        "is_secret": false
-      },
-      {
-        "type": "Basic Auth Credentials",
-        "filename": ".github/workflows/checks.yml",
-        "hashed_secret": "5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8",
-        "is_verified": false,
-        "line_number": 45,
-        "is_secret": false
-      }
-    ],
-    ".github/workflows/daily_checks.yml": [
-      {
-        "type": "Secret Keyword",
-        "filename": ".github/workflows/daily_checks.yml",
-        "hashed_secret": "5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8",
-        "is_verified": false,
-        "line_number": 63,
-        "is_secret": false
-      },
-      {
-        "type": "Basic Auth Credentials",
-        "filename": ".github/workflows/daily_checks.yml",
-        "hashed_secret": "5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8",
-        "is_verified": false,
-        "line_number": 79,
-        "is_secret": false
-      }
-    ],
```

Comment:
Ah, we should get a bit of documentation written up what to do about this.  Instead of adding the one-off comments like below, the thing to do is get the false positives tracked in this file.  You do that by running these two commands at the project root:

```sh
detect-secrets scan --baseline .ds.baseline
detect-secrets audit .ds.baseline
```

The first command will read through everything and update the `.ds.baseline` file, and the second command will walk you through verifying that the things flagged are either a secret or not.

First though, sync with `main` to see if that fixes the issue for you!

If not, please give these commands a try and I can work on getting a bit of documentation written up for it. 🙂